samples/net/sockets/sntp_client/src/main.c

```diff
@@ -57,18 +60,21 @@ void main(void)
 	rv = sntp_init(&ctx, (struct sockaddr *) &addr6,
 		       sizeof(struct sockaddr_in6));
 	if (rv < 0) {
-		LOG_ERR("Failed to init sntp ctx: %d", rv);
+		LOG_ERR("Failed to init SNTP IPv4 ctx: %d", rv);
```

Comment:
I think you have a typo here. Should be SNTP IPv6 right?

Reply:
Indeed, that is a good catch.

Reply:
> Should be SNTP IPv6 right?

Fixed, thanks.